ffs/ffs_file.c

```diff
@@ -1313,7 +1313,7 @@ FILE_INT *file_int_ptr;
 #else
     Baaad shit;
 #endif
-    ntohl(tmp_value);
+    tmp_value = ntohl(tmp_value);
```

Comment:
Is there any particular reason to re-write `tmp_value` here instead of just `*file_int_ptr = ntohl(tmp_value);`
￼

Reply:
No, just interim trying to sort out what was going on.  The original ntohl(tmp_value) was actually a no-op, ignoring the return value.  Ancient cruft.  The ntohl is done at the point of use, rather than in this func.